Resources/public/js/views/services/plugins/ez-versionsplugin.js

```diff
@@ -60,36 +60,14 @@ YUI.add('ez-versionsplugin', function (Y) {
                     api: capi,
                 };
 
-            e.content.loadVersions(options, Y.bind(function (error, versions) {
+            e.content.loadVersionsSortedByStatus(options, function (error, versions) {
                 if (error) {
                     e.target.set('loadingError', true);
                 } else {
                     e.target.set('loadingError', false);
-                    e.target.set('versions', this._sortVersions(versions));
+                    e.target.set('versions', versions);
                 }
-            }, this));
-        },
-
-        /**
-         * Sorts an array of version info by status
-         *
-         * @method _sortVersions
-         * @protected
-         * @param versions {Array} of eZ.VersionInfo
-         * @return {Object} of sorted versions by status:
-         *              struct.<status_name>: {Array} of eZ.VersionInfo
-         */
-        _sortVersions: function(versions) {
```

Comment:
I think this protected method was introduced in 1.4, so you can not removed it, you should instead add a deprecation warning and the `@deprecated` tag so that it's removed in 2.0